docs/managing-mods.md

```diff
@@ -0,0 +1,52 @@
+# Managing mods
+
+> Learn how to manage your mods using Lampray.
+
+## Prerequisites
```

Comment:
Update to: "Install Lampray or build from source" and point to the relevant docs.